### PR TITLE
Removed high z-index of selected guide to enable selection of upper layer elements.

### PR DESCRIPTION
**Pull request by ravishchaudhary**

### CODE REVIEW

File: src/AlignmentGuides.js [hunk 1]
```diff
@@ -684,6 +690,7 @@ class AlignmentGuides extends Component {
 			const id = boxes[box].id || box;
 			const identifier = boxes[box].identifier;  // option index for caption
 			const isSelected = (active === id || activeBoxes.includes(id));
+			id === 'box2' && console.log(active, activeBoxes);
```
Comment (somyarocketium) [inline src/AlignmentGuides.js:693]: @ravishchaudhary can we remove the console log ?

File: src/AlignmentGuides.js [hunk 2]
```diff
@@ -263,7 +263,7 @@ class AlignmentGuides extends Component {
 	dragStartHandler(e, data) {
 		this.setState({
 			active: data.node.id,
-			dragging: true
+			dragging: true,
```
Comment (somyarocketium) [inline src/AlignmentGuides.js:266]: @ravishchaudhary is this extra comma required ?

File: src/AlignmentGuides.js (hunk 1 above)
Comment (ravishchaudhary) [reply src/AlignmentGuides.js:693]: Sure. Will do.

**Comment by ravishchaudhary** [comment]
@somyarocketium Will Re-create with other group related changes.

**Pull request closed without merging**